### DISCUSSION TITLE
Support NETLOGON in DCE_RPC

### DISCUSSION THREAD

**Discussion by ahgit1** [zeek/zeek#1257] [Ideas]
I've noticed I'm getting a lot of dce_rpc_auth_type 68 notices in wierds.log - this appears to be associated with the NETLOGON secure channel auth type. Is there an analyzer available for this and if not should it be logged in wierds or just marked as not supported

thanks

**Comment by rsmmr** [comment]
How would you ideally like to see this logged? Generally we don't have a good place to put such information, which is why they typically end up as weirds. You could enable weird sampling for these if it's mostly about turning down the noise.

**Comment by grigorescu** [comment]
Reading up on NETLOGON a bit, it'd be nice to see this be supported. However, that's not a trivial undertaking.

I'd be willing to go down that path, but probably not by the end of the year. So, we can either leave this in place, or turn it into an Idea for future work.

**Comment by rsmmr** [comment]
I'll turn it into an idea for now.
